platform/master/R/splits.R

```diff
@@ -439,20 +439,7 @@ else{
     }   
   }
 
-  max_argval_size = 0
-  for (av in arg_vals) {
-    max_argval_size = max(max_argval_size, max(unlist(lapply(av, object.size))))
-  }
-  for (rav in raw_arg_vals) {
-    max_argval_size = max(max_argval_size, max(unlist(lapply(rav, object.size))))
-  }
-
-  #this limit applies to each individual argument instead of all arguments together
-  if(max_argval_size >= .rcpp_obj_max_size) {
```

Comment:
There is a variable .rcpp_obj_max_size defined somewhere in the code. Can you pls remove that as well. As it may be confusing to look at it afterwards.